example_school_library_decorator/person.rb

```diff
@@ -4,12 +4,12 @@ class Person
   attr_accessor :name, :age
```

Comment:
- [ ] We also need to refactor the attr_accessor

```suggestion
  attr_accessor :name, :age, :rentals
```